Drivers/Src/CANbus.c

```diff
@@ -104,6 +115,53 @@ static ErrorStatus CANbus_SendMsg(CANId_t id, CANPayload_t payload) {
 	return retVal;
 }
 
+static ErrorStatus CANbus_SendMsg_FaultState(CANId_t id, CANPayload_t payload) {
+	uint8_t txdata[8];
+	uint8_t data_length = 0;
+
+	// TODO: is it really best to keep the list of
```

Comment:
might as well take out the switch case; if we're in the fault state we don't really need this error checking because we can't handle it anyways

Reply:
Wouldn't we need it for the memcpy changes?

Reply:
https://github.com/lhr-solar/Embedded-Sharepoint

Reply:
do nothing for now